### ISSUE TITLE
程序打开/关闭时没有自动打开/关闭Windows系统代理

### ISSUE THREAD

**Issue by howareyouo** [Shadowsocks-NET/Qv2ray#21]
如题。
打开qv2ray时需要手动到设置里开启系统代理，设置端口号。
关闭qv2ray时也会导致所有连接打不开，需要手动设置里关闭代理

**Comment by shanoaice** [comment]
呃……怎么说呢，Qv2ray v3 是没有这个功能的，建议去用 Qv2ray v2 或者 v2rayN

**Comment by howareyouo** [comment]
这是基本的功能，对小白体验很不好啊

> 呃……怎么说呢，Qv2ray v3 是没有这个功能的，建议去用 Qv2ray v2 或者 v2rayN

**Comment by moodyhunter** [comment]
TL;DR: The removal of such features were intended, however, there was a plan to restore them but it's abandoned.

Initially, this repository was a fork of the original version-v3 branch, by me personally. I stripped out many features that I've not been interested in, together with the ones I do not need. I redesigned the project structure (as you can see here: https://github.com/Shadowsocks-NET/Qv2rayBase) and planned new features and fixes for the release of my own version 3.0.

However, I gradually feel it's too complicated and too boring for me to maintain such a large codebase, and what's more, I don't need it, it's actually the demo project I used to learn CMake. So I talked with one of the owners in Shadowsocks-NET, they heavily rebased my commits (thank you BTW) here, which becomes the current repository here.

I have no willingness to update this repository and it seems there isn't someone who, for now, is doing so as well.

So, unfortunately, good luck.

**Comment by howareyouo** [comment]
散了散了，大家一起投奔v2rayN去吧，虽然界面简陋了点，功能和性能还是不输Qv2ray的

**Comment by KevinZonda** [comment]
@howareyouo 草

**Comment by KevinZonda** [comment]
qv，我的qv🤤🤤🤤
